packages/firebase_messaging/firebase_messaging_web/lib/firebase_messaging_web.dart

```diff
@@ -77,33 +73,33 @@ class FirebaseMessagingWeb extends FirebaseMessagingPlatform {
   }
 
   @override
-  Future<RemoteMessage> getInitialMessage() {
+  Future<RemoteMessage?> getInitialMessage() async {
     return null;
   }
 
   @override
-  Future<void> deleteToken({String senderId}) async {
+  Future<void> deleteToken({String? senderId}) async {
     if (!_initialized) {
       // no-op for unsupported browsers
       return;
     }
     try {
-      await _webMessaging.deleteToken();
+      _webMessaging.deleteToken();
     } catch (e) {
       throw utils.getFirebaseException(e);
     }
   }
 
   @override
-  Future<String> getAPNSToken() async {
+  Future<String?> getAPNSToken() async {
     return null;
   }
 
   @override
-  Future<String> getToken({String senderId, String vapidKey}) async {
+  Future<String> getToken({String? senderId, String? vapidKey}) async {
     if (!_initialized) {
       // no-op for unsupported browsers
-      return null;
+      return '';
```

Comment:
This is potentially a breaking change, also iOS/macOS can return a null token as well:

![image](https://user-images.githubusercontent.com/5347038/106606551-2c528680-655a-11eb-8be4-c585b2312874.png)

I think this should probably remain nullable given that on at least 3 platforms it can return null, thoughts?

Reply:
Yep, you're right!